deployment/ccip/shared/stateview/ton/state.go

```diff
@@ -23,6 +23,7 @@ type CCIPChainState struct {
 	CCIPAddress      address.Address
 	OffRamp          address.Address
 	Router           address.Address
+	OnRamp           address.Address
```

Comment:
CCIPAddress is OnRamp (let's remove CCIPAddress and keep OnRamp), it was just copied from Aptos since the code is structured as a CCIP package there